### DISCUSSION TITLE
[委托]小垃圾也要追求爱情！

### DISCUSSION THREAD

**Discussion by Deng416** [share121/inter-knot#679] [其他]
虽然第一次纪念日过得有些狼狈，但我可爱的女朋友完全完全完全原谅了我！
如果是她的话，哪怕每年纪念日都让我去一次空洞，我也一定心甘情愿！
虽然我一事无成，过着垃圾一样的人生，但这样卑微的生活，其实也有闪光之处！
我有一位非常可爱的女朋友，但因为太可爱了，我自己知道就好，所以就不放图片了。
我们马上要在一起一年了！
最后，我现在有一个小小小问题，我的女友希望一周年纪念日要在初次心动的地方度过，但我们初次见面是在空洞内避难···
所以，有没有人可以把我送进空洞，带我去和女友的初见地点。
拜托了！我已经下定决心，哪怕堵上性命也要完美度过这一次的恋爱纪念日！
求求谁来帮我这个小垃圾一把，让我圆梦！

**Comment by langxmo** [comment]
急急急，三秒内，我要看到可爱女友的照片！不发照片的一律按臆想处理！
